WordPress/Classes/Utility/Router.swift

```diff
@@ -0,0 +1,112 @@
+
+import WordPressComStatsiOS
+
+/// `Router` is intended to be used to easily navigate and display common elements natively
+/// like Posts, Site streams, Comments, etc...
+///
+struct Router {
```

Comment:
I think it would be a good idea to abstract this behind a protocol, so that we can mock it in tests. What do you think?

Reply:
Sounds good!
What do you think about the name?

Reply:
Maybe it's a little bit too generic? I mean, even though it's a Router, it seems a bit specific to notifications / activity log, and there is already an app-wide Router. I am not sure what name to suggest though, I am terrible at that. Maybe `NotificationRouter` or `FormattableContentRouter`?

Reply:
What about simply `ContentRouter`, "to display content on their native elements".

Reply:
So the classes I introduced for navigation, I went with the term `Coordinator` to roughly follow along the idea of coordinators popularised by Soroush Khanlou: http://khanlou.com/2015/10/coordinators-redux/

> A coordinator is an object that bosses one or more view controllers around. Taking all of the driving logic out of your view controllers, and moving that stuff one layer up

But don't feel you have to go with that! :)